e2e-tests/cypress/tests/integration/channels/bot_accounts/bot_api_2_spec.ts

```diff
@@ -180,7 +183,7 @@ describe('Bot accounts ownership and API', () => {
                 // # Create a post
                 cy.postBotMessage({channelId: channel.id, message: msg2, token, failOnStatus: false}).then(({status}) => {
                     // * Validate that posting failed
-                    expect(status, 403);
+                    expect(status).to.equal(403);
```

Comment:
This should be `401` since the token is no longer valid and the request became unauthorized.
```suggestion
                    expect(status).to.equal(401);
```